### PR TITLE
Added ibm-watson-iot org

### PR DESCRIPTION
**Pull request by daphoenix**
Added ibm-watson-iot org for our open source projects to be aggregated and displayed under IBM.io site.

### CODE REVIEW

File: orgs.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 
     var orgs = [
+        {"name": "ibm-watson-iot",
+         "type": "repo"},
```
Comment (stevemart) [inline orgs.js]: type should be 'org' here, since you want the entire https://github.com/ibm-watson-iot org to be added, not just a single repository

Comment (daphoenix) [reply orgs.js]: OK. That was a typo.

**Comment by daphoenix** [comment]
Made the changes @stevemart .

**Comment by stevemart** [comment]
thanks!

**Pull request merged**